<launch>
	<node name="fork_grasps" pkg="agile_grasp" type="find_grasps" output="screen">
  	<param name="cloud_topic" value="/fork/pcl_seg/pcl" />
    <param name="cloud_frame" value="/camera_rgb_optical_frame" />
    <param name="cloud_type" value="0" />
    <param name="svm_file_name" value="$(find agile_grasp)/svm_032015_20_20_same" />
    <param name="num_samples" value="4000" />
    <param name="num_threads" value="4" />
    <param name="num_clouds" value="1" />
		<rosparam param="workspace"> [-10, 10, -10, 10, -10, 10] </rosparam>
    <rosparam param="camera_pose"> [0, 0.445417, 0.895323, 0.215, 
              1, 0, 0, -0.015, 
              0, 0.895323, -0.445417, 0.23, 
              0, 0, 0, 1] </rosparam>
    <param name="min_inliers" value="10" />
    <param name="plotting" value="2" />
    <param name="marker_lifetime" value="5" />
	</node>
</launch>
